.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,62 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
```

Comment:
My eyes were spinning too much, so I split the workflows into multiple files that I could compare with `diff` without going bananas.

Tried using anchors, but they aren't supported by GitHub's server-side parser.